### PR TITLE
{vis}[intel-2016a] Pillow 3.2.0 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-intel-2016a-Python-2.7.11.eb]: @hajgato do you mind reworking this to use the `%(pyver)s` templates we support now?

Something like:

```python
name = 'Pillow'
version = '3.2.0'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2107e22e2315a11d5186d8ef57e634cd for a full test report.

**Comment by hajgato** [comment]
@boegel Do we also have `%(pyshortver)s` ?

**Comment by boegel** [comment]
@hajgato yes, we do:

```
$ eb --avail-easyconfig-templates
...
Template names/values for (short) software versions
    %(javashortver)s: short version for Java (<major>.<minor>)
    %(javaver)s: full version for Java
    %(perlshortver)s: short version for Perl (<major>.<minor>)
    %(perlver)s: full version for Perl
    %(pyshortver)s: short version for Python (<major>.<minor>)
    %(pyver)s: full version for Python
    %(rshortver)s: short version for R (<major>.<minor>)
    %(rver)s: full version for R
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node2621.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3be1b357002be70c206330648142ef33 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**